src/lib/etasu.ts

```diff
@@ -33,13 +33,19 @@ router.get('/met/:caseId', async (req: Request, res: Response) => {
   res.send(await remsCaseCollection.findOne({ case_number: req.params.caseId }));
 });
 
+router.get('/met/auth/:authNumber', async (req: Request, res: Response) => {
```

Comment:
I see you added a method to search for the ETASU (REMS Case) by using the Auth Number. This interface is unlikely to get used though. Should we add a way to search for ETASU using FHIR? Maybe handle the case where a REMS case number is passed in as an input parameter to the operation.

Reply:
I think we already have endpoints to look up ETASU by the case number and also patient demographics, thought wouldn't hurt to check that they didn't get removed at some point.